manifests/base/applicationset-controller/argocd-applicationset-controller-role.yaml

```diff
@@ -48,20 +48,30 @@ rules:
       - patch
       - watch
   - apiGroups:
-      - ''
+      - ""
     resources:
       - secrets
       - configmaps
     verbs:
       - get
       - list
       - watch
+  # argocd-applicationset-controller leader election rules
+  # Create with resourceNames fails, so use a separate rule for the lease creation
   - apiGroups:
-      - apps
-      - extensions
+      - coordination.k8s.io
     resources:
```

Comment:
I am confused why the applicationset controller needs get, list, watch for deployments. I removed it but can add it back.